src/pages/Home/home.js

```diff
@@ -1,7 +1,43 @@
-export const Home = () => {
-  return (
-    <>
-      New Home
-    </>
-  )
-}
\ No newline at end of file
+import axios from "axios";
+import React from "react";
+
+export class Home extends React.Component {
+  state = {
+    data: {},
+  };
+  getBooksByGenre = async (subject) => {
+    try {
+      const { data } = await axios(
+        `https://www.googleapis.com/books/v1/volumes?q=subject:${subject}&maxResults=40&key=${process.env.REACT_APP_API_KEY}`
+      );
+      this.setState({ data: data.items });
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  componentDidMount() {
+    this.getBooksByGenre("horror");
+  }
+
+  MappedData = () => {
+    const mappedData = this.state.data.map((object) => {
+      return (
+        <div>
+          {object.volumeInfo.title}
+          <div>
+            <img
+              src={`${object.volumeInfo.imageLinks.thumbnail}`}
+              alt="book cover"
+            />
+          </div>
+        </div>
+      );
+    });
+    return <li>{mappedData}</li>;
+  };
+
+  render() {
+    return <>{this.state.data.length && <div>{this.MappedData()}</div>}</>;
```

Comment:
I think you dont need to nest the constent inside the curly braces in a nameless component, it just renders a component when a condition is met.

Reply:
I tried without and it didn't recognize the variables.

Reply:
> I think you dont need to nest the constent inside the curly braces in a nameless component, it just renders a component when a condition is met. So the result will be a component when displayed

can you leave an example here with what do you mean?

Reply:
Here I would ask why did you use uppercase for `MappedData`

Reply:
I think because originally  it was going to be a component, but now I understand it is just a variable, right?